### PR TITLE
exercise-loops-and-functions

### PR DESCRIPTION
**Pull request by nkoblai**

### CODE REVIEW

File: go-tour/loops-and-functions.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+//Sqrt returns a square root of number
+func Sqrt(x float64) float64 {
+	z := 1.0
+	for n := 0; n <= 10; n++ {
+		z = z - ((z*z - x) / (2 * z))
+	}
+	return z
+}
+
+//Implement the square root function using Newton's method
```
Comment (arbrix) [inline go-tour/loops-and-functions.go:17]: `square root function using Newton's method` should be at the comment of function, what do you think?

Comment (nkoblai) [reply go-tour/loops-and-functions.go:17]: Agree, but i think comment should be `Sqrt returns a square root of number using Newton's method`. It clearly describes the fuctionality of the function.

Comment (arbrix) [reply go-tour/loops-and-functions.go:17]: agree

Comment (arbrix) [reply go-tour/loops-and-functions.go:17]: please review this [instruction](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences)

**Review by arbrix — APPROVED** [review]
LGTM

**Pull request merged**